docs(postman): Update documentation for postman tests (#2057)
juspay/hyperswitch: crates/test_utils/README.md

# Test Utils

> The heart of `newman-dir` and `UI-tests`

Author: I preferred giving the fork a name rather than using `newman` since it does not differentiate both, so used `newman-dir` as a reference everywhere 😅

# Test Utils

The heart of `newman-dir` and `UI-tests`

## Usage

- Make sure you that you've **do not** have the official newman installed but rather `newman-dir`

>   - `newman-dir` can be installed by running `npm install -g 'git+ssh://git@github.com:knutties/newman.git#feature/newman-dir'`

Reviewer: ```suggestion
  - `newman-dir` can be installed by running `npm install -g 'https://github.com/knutties/newman.git#feature/newman-dir'`
```

Something like this should also work.

>   ```doc

Reviewer: `text` should also be fine.

```suggestion
  ```text
```

  dir-create [options] <collection-path>                      Create a directory based Postman collection in the given path
  dir-export [options] <postman-collection-file>              Convert a Postman collection file into its directory representation
  dir-export-import-test [options] <postman-collection-file>  Check if an export followed by import results in same collection
  dir-import [options] <collection-dir>                       Convert a Postman directory representation into a postman collection
  dir-remove-folder <folder-path>                             Remove test at given path from directory based Postman collection
  dir-remove-test <test-path>                                 Remove test at given path from directory based Postman collection
  dir-run [options] <collection-dir>                          Runs the tests in collection-dir, with all the provided options
  run [options] <collection>                                  Initiate a Postman Collection run from a given URL or path 
  ```
- Add the connector credentials to the `connector_auth.toml` / `auth.toml`
- Export the auth file path as an environment variable:

>   ```zsh

Reviewer: Prefer using `shell` instead of `bash` or `zsh`, here and other usages.

```suggestion
  ```shell
```
